create-release.py

```diff
@@ -438,8 +438,14 @@ def build_release():
     print("----------------------- Building Release ------------------------")
     print("-----------------------------------------------------------------")
 
+    # Build wheels and source packages
     check_run(["make", "release"], cwd=config.source_dir, capture_output=False)
 
+    # Build container images from tagged release
+    check_run(["git", "checkout", f"tags/v{config.new_version}"], cwd=config.source_dir, capture_output=False)
+    check_run(["make", "container-images"], cwd=config.source_dir, capture_output=False)
+    check_run(["git", "checkout", "main"], cwd=config.source_dir, capture_output=False)
+
```

Comment:
the checkouts are handled before this, on the main driver...
you just need to build the images here

Reply:
I think up to this point, the current branch of the repository under `build/release/elyra` would be set to `main` still. So any release built with these checkouts would be built using the `dev` tag

Reply:
@lresende - if youre good with this im going to merge